doc/topics/releases/2018.3.2.rst

```diff
@@ -0,0 +1,16 @@
+===========================
+In Progress: Salt 2018.3.2 Release Notes
+===========================
+
+Version 2018.3.2 is an **unreleased** bugfix release for :ref:`2018.3.0 <release-2018-3-0>`.
+This release is still in progress and has not been released yet.
+
+Changes to win_timezone
+-----------------------
```

Comment:
Please change this underline to use `====` instead of `----`, to match the pending changes to the changelog generator. The order of use of different underline chars affects the section names generated by Sphinx.